.github/workflows/ci_cron_daily.yml

```diff
@@ -38,8 +38,7 @@ jobs:
     with:
       submodules: false
       coverage: codecov
-      toxdeps: |
-        tox < 4
+      toxdeps: "<4"
```

Comment:
I guess you need to pairs of quotes. e.g. try `"'4'"`, one makes it a yaml string, one must be insterted into the shell

Reply:
The error mentioned tox twice, so I omitted "tox"

Reply:
tox twice is not an issue, `pip install -U tox "tox<4"` installs tox 3 for me